pyfar/dsp/fft.py

```diff
@@ -224,7 +224,7 @@ def normalization(spec, n_samples, sampling_rate, fft_norm='none',
     Note that the phase is maintained in all cases, i.e., instead of taking
     the squared absolute values for ``'power'`` and ``'psd'``, the complex
     spectra are multiplied with their absolute values to ensure a correct
-    renormalization. Details on the FFT normalization given by `fft_norm` are
+    renormalization. Details on the FFT normalization given by ``fft_norm`` are
```

Comment:
Variables should be italic according to our guidelines: https://pyfar-gallery.readthedocs.io/en/latest/contribute/contribution_packages.html#writing-the-documentation
```suggestion
    renormalization. Details on the FFT normalization given by `fft_norm` are
```

Reply:
upps, thank you, looks like I did it wrong all the time